OpenKh.Tests/Bbs/PmoTests.cs

```diff
@@ -0,0 +1,138 @@
+using OpenKh.Common;
+using OpenKh.Bbs;
+using System.IO;
+using Xunit;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System;
+
+namespace OpenKh.Tests.Bbs
+{
+    public class PmoTests
+    {
+        private static readonly string FileName = "Bbs/res/p01ex00.pmo";
```

Comment:
I have a change request about this, but I will comment it on Discord for safety reasons.